src/index.jsx

```diff
@@ -1,11 +1,32 @@
 import 'core-js/fn/object/assign';
 import React from 'react';
 import ReactDOM from 'react-dom';
+// Import routing components
+import { Router, Route, browserHistory } from 'react-router';
 
-import App from './components/app';
+var data = [
+	{
+	info1: 'Tired',
+	info2: 'More Awake'
+	},
+	{
+	info1: 'Very Tired',
+	info2: 'Even More Awake'
+	},
+	{
+	info1: 'Sleeping',
+	info2: 'Buzzin'
+	}]
+
+import Home from './components/Pages/Home.jsx';
+import About from './components/Pages/About.jsx';
+import Training from './components/Pages/Training.jsx';
 
-// Render the main component into the dom
 ReactDOM.render(
-  <App />,
-  document.getElementById('app')
+    <Router history={browserHistory}>
+        <Route path="/" component={Home}/>
+        <Route path="/About" component={About} data={data}/>
+        <Route path="/Training" component={Training} data={data}/>
+    </Router>,
```

Comment:
This version of react-router does some interesting things with nesting routes, which is cool, but they seem to have removed that in later versions, preferring to compose the pages (as you've done) so I'll leave this be.